.github/workflows/lint-test.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         uses: astral-sh/setup-uv@v7
         with:
           enable-cache: true
+          resolution-strategy: "lowest"
```

Comment:
What is the motivation behind this in ci only?

Reply:
Where else would it be? It's a parameter for the action, not for uv itself (despite uv having a identically named parameter).

https://github.com/astral-sh/setup-uv/blob/30ce38e20653a26dd16caa082b010cec334ccf56/src/download/download-version.ts#L180-L183

We want to:
- Always use the lowest confirmed working version of `uv` in CI (for us it'll use `0.9.7` since our specifier is `>=0.9.7`)
- Simultaneously, we don't want to annoy people developing locally with newer versions of uv by adding an upper bound. We don't want CI to magically switch to 0.10.0 but it doesn't matter if we do locally.

Reply:
Ah, thanks! Its a slightly confusingly named parameter.